pebble/src/test/resources/data/sampleContext.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Maybe we could unify handlebars and pebble tests data (in the future)? It does not have to be in this PR, but it could show that similar things can be achieved by both.